functions/index.js

```diff
@@ -43,12 +43,41 @@ exports.matchMeNow = functions.https.onRequest((request, response) => {
                 let currentUserZipCode = currentUserDoc.data().zipCode;
                 let currentUsergenderPreference = currentUserDoc.data().genderPreference;
                 let currentUserLanguage = currentUserDoc.data().language;
```

Comment:
Would users only be allowed to enter one language or would it be an array? if it was an array we would have to update the `array-contains` function to `array-contains-any`

Reply:
In the google doc description of data structure it says string for user language. I can change it to an array to keep it flexible